lib/new_relic/agent/instrumentation/concurrent_ruby/chain.rb

```diff
@@ -0,0 +1,23 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module ConcurrentRuby
+    def self.instrument!
+      # this is a module, can I still use class_eval?
```

Comment:
Hmm... this should be updated with the new class. I think something's wrong with my pushes.